lib/web/session/cookie.go

```diff
@@ -84,6 +84,10 @@ func ClearCookie(w http.ResponseWriter) {
 }
 
 const (
-	// CookieName is the name of the session cookie.
+	// CookieName is the name of the Web UI session cookie.
 	CookieName = "__Host-session"
+
+	// SAMLSessionCookieName is the name of the SAML IdP session cookie.
+	// This cookie is set by SAML IdP after a successful SAML authentication.
+	SAMLSessionCookieName = "__Host-saml_session"
```

Comment:
Similarly as above, let's avoid mixing cookies with different purposes in this file.